tests/kafkatest/benchmarks/streams/streams_simple_benchmark_test.py

```diff
@@ -48,6 +48,7 @@ def test_simple_benchmark(self, test, scale):
         # SETUP PHASE
         #############
         self.zk = ZookeeperService(self.test_context, num_nodes=1)
+        self.zk.log_level = "INFO"
```

Comment:
It doesn't look like this does anything -- afaict the `ZookeeperService` doesn't currently render the `log4j.properties` file and it won't use the same one as Kafka since it's on its own node.